File: 2.0/python/numbers/getNumberLeaseConfig.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.numbers.getNumberLeaseConfig(
    number='12132000384',
    # return only specific fields
    fields='number,configType'
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/numbers/getNumberLeaseConfig.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->getNumberLeaseConfig();
        $request->getOperationConfig()->setPathParameters(array("number" => "12132000384"));
        $request->getOperationConfig()->setQueryParameters(array("fields" => "number,configType"));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
